Document/Document.ipynb

```diff
@@ -1228,6 +1229,112 @@
     "</ul> "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Iterating and casting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From `version 3.5`, `ConfusionMatrix` is an **iterator** object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0 {0: 3, 1: 0, 4: 0}\n",
+      "1 {0: 0, 1: 1, 4: 0}\n",
+      "4 {0: 0, 1: 0, 4: 0}\n"
+     ]
+    }
+   ],
+   "source": [
+    "for row,col in cm:\n",
```

Comment:
`for row, col in cm:`

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/428/commits/488970a6ffcaceee93c3ab4f0dc8d6732bccf0ed